.travis.yml

```diff
@@ -5,20 +5,35 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
+  - 2.6.2
   - ruby-head
+  - jruby
```

Comment:
Should this really be using jruby? Or does this test with all the rubies listed?

Reply:
Yes this test with all the Rubies listed. The idea being to try to see where the gem will run. The places which are not expected to always succeed are marked as 'allow_failure' so the will not fail the build. (i.e., jruby and ruby-head)